spark-tsfile/src/test/scala/org/apache/iotdb/tsfile/TSFileSuit.scala

```diff
@@ -214,4 +214,28 @@ class TSFileSuit extends FunSuite with BeforeAndAfterAll {
     Assert.assertEquals(expected, df.schema)
   }
 
+  test("testTransform1") {
+    val df = spark.read.tsfile(tsfile1)
+    df.createOrReplaceTempView("tsfile_table")
+    val newDf = spark.sql("select * from tsfile_table " +
+      "where (`device_1.sensor_1`>0 or `device_1.sensor_2` < 22) and time < 4")
+    Assert.assertEquals(3, newDf.count())
+  }
+
+  test("testTransform2") {
```

Comment:
test("testTransform2")  and test("testTransform3") can't pass.
Please check this problem.